src/vmm/src/devices/virtio/block/device.rs

```diff
@@ -15,6 +14,7 @@ use crate::devices::virtio::device::VirtioDevice;
 use crate::devices::virtio::queue::Queue;
 use crate::devices::virtio::{ActivateError, TYPE_BLOCK};
 use crate::rate_limiter::BucketUpdate;
+use crate::snapshot::Persist;
```

Comment:
commit `refactor(block): replace persist type aliases`: would be good to have more explanation of the motivation.

Reply:
updated commit message